doc/decisions/0008-remove-final-summary-confirmation-blocks.md

```diff
@@ -0,0 +1,66 @@
+# 7. Remove final summary and confirmation blocks
+
+## Context
+
+Linear non-hub questionnaires must either end with a `Summary` or `Confirmation` block. This block determines what a user sees after completing and before submitting a questionnaire. As the block must be included as the last block in the questionnaire, the block is either included in:
+- the last group
+- an extra group added for the summary block
+- an extra section (with one group) added for the summary block
+
+This is not ideal for two reasons:
+- The block is related to the whole questionnaire, not any given group or section
+- Any time we condsider completeness for a given section, we need to consider if a `Summary` or `Confirmation` block exists.
+
+The `SectionSummary` block type was [removed](https://github.com/ONSdigital/eq-questionnaire-validator/pull/29) for similar reasons. 
+
+## Proposal
+
+- Remove the `Summary` and `Confirmation` block types
+- Introduce a new `questionnaire_flow` top-level schema object with:
+  - a `type` property to differentiate between `Linear` and `Hub` questionnaires
+  - an `options` object to hold options related to the given questionnaire flow:
+    - For `Linear` questionnaires, an `include_summary_on_submission` boolean option
```

Comment:
This should just be `inculde_summary`?. There is no other place this applies to and to me, it reads like it is on the submission click i.e. after submission rather than on final page.

Reply:
I agree with @MebinAbraham - the `include_summary_on_submission` does lean towards suggesting it will be on submission click. I think `include_summary`, given the other constraints set out here, is enough.

Reply:
I did have that originally. I'll change it back 👍